src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -0,0 +1,129 @@
+        }),
+    }),
+
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
+    iou: PropTypes.shape({
+        id: PropTypes.string,
+        amount: PropTypes.number,
+        currency: PropTypes.string,
+        participants: PropTypes.arrayOf(participantPropTypes),
+    }),
+
+    /** Which tab has been selected */
+    selectedTab: PropTypes.string,
+};
+
+const defaultProps = {
+    route: {
+        params: {
+            iouType: '',
+            reportID: '',
+        },
+    },
+    iou: {
+        id: '',
+        amount: 0,
+        currency: CONST.CURRENCY.USD,
+        participants: [],
+    },
+    selectedTab: CONST.TAB.MANUAL,
+};
+
+function MoneyRequestSelectorPage(props) {
+    const iouType = lodashGet(props.route, 'params.iouType', '');
+    const reportID = lodashGet(props.route, 'params.reportID', '');
+    const {translate} = useLocalize();
+    const {canUseScanReceipts} = usePermissions();
+
+    const title = {
+        [CONST.IOU.MONEY_REQUEST_TYPE.REQUEST]: translate('iou.requestMoney'),
+        [CONST.IOU.MONEY_REQUEST_TYPE.SEND]: translate('iou.sendMoney'),
+        [CONST.IOU.MONEY_REQUEST_TYPE.SPLIT]: translate('iou.splitBill'),
+    };
+
+    const resetMoneyRequestInfo = () => {
+        const moneyRequestID = `${iouType}${reportID}`;
+        IOU.resetMoneyRequestInfo(moneyRequestID);
+    };
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            {({safeAreaPaddingBottomStyle}) => (
+                <FullPageNotFoundView shouldShow={!IOUUtils.isValidMoneyRequestType(iouType)}>
+                    <DragAndDropProvider isDisabled={props.selectedTab === CONST.TAB.MANUAL}>
+                        <View style={[styles.flex1, safeAreaPaddingBottomStyle]}>
+                            <HeaderWithBackButton
+                                title={title[iouType]}
+                                onBackButtonPress={Navigation.dismissModal}
+                            />
+                            {canUseScanReceipts && iouType === CONST.IOU.MONEY_REQUEST_TYPE.REQUEST ? (
+                                <OnyxTabNavigator
+                                    id={CONST.TAB.RECEIPT_TAB_ID}
+                                    tabBar={({state, navigation}) => (
+                                        <TabSelector
+                                            state={state}
+                                            navigation={navigation}
+                                            onTabPress={resetMoneyRequestInfo}
+                                        />
+                                    )}
+                                >
+                                    <TopTab.Screen
+                                        name={CONST.TAB.MANUAL}
+                                        component={MoneyRequestAmount}
+                                        initialParams={{reportID, iouType}}
```

Comment:
Hi all, in this PR we broke the functionality where you set currency in the request param.

eg: `/request/new?currency=CHF`

https://github.com/Expensify/App/pull/23636

Reply:
Oops, it looks like this is resolved now?